src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -34,6 +35,7 @@ async function run() {
         let dataMirror: string | undefined = tl.getInput('dataMirror');
         let customRepo: string | undefined = tl.getInput('customRepo');
         let additionalArguments: string | undefined = tl.getInput('additionalArguments');
+        let hasLocalInstallation = true;
```

Comment:
Will tell us if a local installation/custom path of the dependency check tool is used (true) or not (false)